src/components/cluster/Cluster.tsx

```diff
@@ -307,8 +313,8 @@ function Cluster({
                             </div>
                             {clusterId && <List.DropDown src={<Pencil color="#b1b7bc" onClick={handleEdit} />} />}
                         </List>
-                        {serverMode !== SERVER_MODE.EA_ONLY && clusterId ? <hr className="mt-0 mb-0" /> : null}
-                        {serverMode !== SERVER_MODE.EA_ONLY && clusterId ? (
+                        {serverMode !== SERVER_MODE.EA_ONLY && !window._env_.K8S_CLIENT && clusterId ? <hr className="mt-0 mb-0" /> : null}
+                        {serverMode !== SERVER_MODE.EA_ONLY && !window._env_.K8S_CLIENT && clusterId ? (
```

Comment:
can we combine these 2? same checks are performed twice or move `this.props.serverMode === SERVER_MODE.EA_ONLY || window._env_.K8S_CLIENT` to top & make use of it.

Reply:
done